src/env.d.ts

```diff
@@ -3,6 +3,7 @@
 declare const webpackEnv: {
     readonly target: 'Chromium' | 'Firefox' | 'WKWebview' | undefined
     readonly firefoxVariant: 'android' | 'desktop' | 'GeckoView' | undefined
+    readonly version: string
```

Comment:
🤷‍♂️

Reply:
What is wrong with using string for version? 🤔

Reply:
@neruthus there are nothing wrong with the type but I'm not encouraging to use it as a webpack-defined constant